appengine/flexible/ruby31-and-earlier/analytics/app.rb

```diff
@@ -0,0 +1,47 @@
+# Copyright 2016 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START gae_flex_analytics_track_event]
```

Comment:
I'm not too sure about this. I'm under the impression that these tags have to be unique across languages or apps. I'm seeing this in mutilple apps. So, if you are also uncertain about this, please feel free to ask the Docs team.

Reply:
Correction: I think DevRels team enforces that rule. So, maybe confirm it with someone in Jennifer's team.

Reply:
The tags do not have to be unique. We add tags so that we can show a snippet from a file instead of the entire file. We always use a combination of file name + tag in the docs, which is unique.